projects/RabbitMQ.Client/RabbitMQ.Client.csproj

```diff
@@ -33,6 +33,7 @@
 
   <PropertyGroup>
     <AnalysisMode>AllEnabledByDefault</AnalysisMode>
+    <EnableNETAnalyzers>false</EnableNETAnalyzers>
```

Comment:
This would a way to globally disable the analyzers, though since this project isn't targeting `net5.0`, they are already disabled by default. Seems like this is redundant at the moment.

Reply:
I removed this line so they aren't all disabled now.